package.json

```diff
@@ -3,7 +3,6 @@
   "version": "1.0.1",
   "description": "Template for nodejs with unit gulp build, test, coverage, code guardian, github and Other defaults",
   "main": "index.js",
-  "type": "module",
```

Comment:
We can still use modules with require. So to convert something like `require("path")` to module format, just use `import path from "path";` with the module syntax.